Migrating-AU-to-Chocolatey-AU.md

```diff
@@ -0,0 +1,29 @@
+# Migrating From `AU` To `Chocolatey-AU`
+
+With the [`1.0.0` release](https://github.com/chocolatey-community/chocolatey-au/releases/tag/1.0.0) of `Chocolatey-AU`, there is a need to migrate to the new package name.
+
+How you migrate from `AU` to `Chocolatey-AU` will depend on how you used `AU` in your repository. Below are some scenarios, but if you encounter a scenario not yet covered, please reach out in the [Chocolatey Community](https://ch0.co/community).
+
+
+## Your Repository Currently Clones the Original `AU` Repository

```

Comment:
```suggestion
## Your Repository Currently Forks the Original `AU` Repository
```

Reply:
Most likely not the appropriate change. Clones would still be correct here.

Forking the repository would be creating a copy of the repository into a new repository, but most likely they are just taking the original repository and cloning it to a local instance (technically, a fork is also a clone, but a clone is not necessarily a fork).

Reply:
Good catch. This was based off of some discussions in Discord. I'll add some clarifications about the cloning part to make it clearer.